Source/WebCore/css/CSSValueList.cpp

```diff
@@ -218,10 +218,7 @@ CSSValueListBuilder CSSValueContainingVector::copyValues() const
 
 void CSSValueContainingVector::serializeItems(StringBuilder& builder) const
 {
-    auto prefix = ""_s;
-    auto separator = separatorCSSText();
-    for (auto& value : *this)
-        builder.append(std::exchange(prefix, separator), value.cssText());
+    builder.append(interleave(*this, [](auto& value) { return value.cssText(); }, separatorCSSText()));
```

Comment:
So irritating that this creates so many `String` temporaries.

Reply:
I’ve been pretty consistently adding both StringBuilder and non-StringBuilder versions for al the color value work, and we should propagate that idiom to all values.